src/cli/core/GeneratedFileWriter.ts

```diff
@@ -86,24 +87,48 @@ export class GeneratedFileWriter {
     const contentGenerator = new FileContentGenerator(generatedClassnames, scanner.getPrefix());
     const fileContentTemplate = contentGenerator.generateFileContent();
 
+    // Resolve the custom classes import path relative to the output file
+    let customClassesImportPath: string | null = null;
+    if (!!this._outputFilename && !!this._customClassesFilename) {
+      customClassesImportPath = path
+        .join(
+          path.relative(
+            path.join(process.cwd(), path.dirname(this._outputFilename)),
+            path.join(process.cwd(), path.dirname(this._customClassesFilename)),
+          ),
+          path.basename(this._customClassesFilename),
+        )
+        // Convert any Windows path separators to posix
+        .replace(/\\/g, '/')
+        .replace(/(\.d)?\.ts$/, '');
+      customClassesImportPath =
+        customClassesImportPath[0] === '.'
+          ? customClassesImportPath
+          : `./${customClassesImportPath}`;
+    }
+
```

Comment:
To generate a proper import path, I compute a relative path between the `outputFilename` and the `customClassesFilename`.

This results in an import like the following:
```ts
import type TCustomClassesFromExternalFile from './tailwindcss-classnames-custom-classes';
```